apps/remix/app/routes/$.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  json,
+  type LoaderFunctionArgs,
+  type MetaFunction,
+} from '@remix-run/node';
+import {
+  dehydrate,
+  QueryClient,
+  HydrationBoundary,
+  useQuery,
+} from '@tanstack/react-query';
+import { flattenToAppURL } from '../utils';
+import { useLoaderData, useLocation } from '@remix-run/react';
+import { usePloneClient } from '@plone/providers';
+import { Breadcrumbs, RenderBlocks } from '@plone/components';
+import config from '@plone/registry';
+import { ploneClient } from '../client';
+
+export const meta: MetaFunction = () => {
+  return [
+    { title: 'New Remix App' },
+    { name: 'description', content: 'Welcome to Remix!' },
```

Comment:
Perhaps brand the above two lines with Plone?

Reply:
done